features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,634 @@
+      | naam                        | waarde   |
+	  | land.code                   | 0000     |
+	  | land.omschrijving           | Onbekend |
+      | datumAanvangAdresBuitenland | 20080402 |
+
+  Rule: wanneer gevraagd wordt om vanuitVerblijfplaatsOnbekend, wordt landVanwaarIngeschreven geleverd
+
+    Scenario: met fields is gevraagd om vanuitVerblijfplaatsOnbekend
+      Gegeven de persoon met burgerservicenummer '000000358' heeft de volgende 'verblijfplaats' gegevens
+      | naam                                 | waarde      |
+      | land vanwaar ingeschreven (14.10)    | 5003        |
+      | datum vestiging in Nederland (14.20) | 19750101    |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000358                       |
+      | fields              | vanuitVerblijfplaatsOnbekend    |
+      Dan heeft de response een persoon met de volgende 'immigratie' gegevens
+      | naam                                 | waarde      |
+      | landVanwaarIngeschreven.code         | 5003        |
+      | landVanwaarIngeschreven.omschrijving | Zwitserland |
+
+  Rule: wanneer gevraagd wordt om indicatieVestigingVanuitBuitenland, wordt datumVestigingInNederland geleverd
+
+    Scenario: met fields is gevraagd om indicatieVestigingVanuitBuitenland
+      Gegeven de persoon met burgerservicenummer '000000371' heeft de volgende 'verblijfplaats' gegevens
+      | naam                                 | waarde      |
+      | land vanwaar ingeschreven (14.10)    | 5003        |
+      | datum vestiging in Nederland (14.20) | 19750101    |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 000000371                          |
+      | fields              | indicatieVestigingVanuitBuitenland |
+      Dan heeft de response een persoon met de volgende 'immigratie' gegevens
+      | naam                      | waarde   |
+      | datumVestigingInNederland | 19750101 |
+
+  Rule: wanneer gevraagd wordt om een adresregel1, adresregel2, adresregel3 of land in de adressering, dan worden de verblijfplaats velden geleverd die nodig zijn voor het samenstellen daarvan
+    - Veld adresregel1 wordt samengesteld uit straat + aanduidingBijHuisnummer + huisnummer + huisletter + huisnummertoevoeging + locatiebeschrijving + regel 1 adres buitenland
+    - Veld adresregel2 wordt samengesteld uit postcode + woonplaats + gemeente van inschrijving + regel 2 adres buitenland
+    - veld adresregel3 wordt gevuld met regel 3 adres buitenland
+    - veld land wordt gevuld met land
+
+    Abstract Scenario: met fields is gevraagd om <veld> van een <adressoort>
```

Comment:
Ook hier de volgende foutmelding `duplicate key value violates unique constraint "lo3_adres_idx_0"`

Reply:
Deze gaat nu fout (de insert)  omdat <gemeentecode> en <gemeenteVanInschrijving> verwisseld zijn. <gemeentecode> moet bij het adres (want altijd verplicht in lo3_adres)